server/utils/bedUtils.ts

```diff
@@ -70,3 +72,25 @@ const bedLink = (bed: BedSummary, premisesId: string): string =>
       attributes: { 'data-cy-bedId': bed.id },
     },
   )
+
+export const encodeOverbooking = (overbooking: BedOccupancyOverbookingEntryUi): string => {
+  const json = JSON.stringify(overbooking)
+
+  return Buffer.from(json).toString('base64')
+}
+
+export const decodeOverbooking = (string: string): BedOccupancyOverbookingEntryUi => {
+  const json = Buffer.from(string, 'base64').toString('utf-8')
+  const obj = JSON.parse(json, (name, value) => {
+    if (['startDate', 'endDate'].includes(name)) {
```

Comment:
Maybe I'm being overcautious but would it be worth putting all of this in a `try/catch` in case any of it blows up?

Reply:
I think we'd want it to blow up with an invalid date exception if there was something wrong with the input.